client/app/lib/components/sidebarstacksection/index.coffee

```diff
@@ -150,48 +150,63 @@ module.exports = class SidebarStackSection extends React.Component
     return  if MENU
 
     callback = @bound 'onMenuItemClick'
-
     menuItems = {}
-    template = @props.template
 
-    { originId, accessLevel } = template.toJS()  if template
+    { template, stack } = @props
+    { storage } = kd.singletons.computeController
+
+    template = template.toJS()
+    stack = stack.toJS()
+    account = whoami()
+
+    isMyTemplate = template?.originId is account._id
```

Comment:
we can use `template.isMine()` here but probably you need to use `remote.revive` for that, just saying.